chart/kubeapps/templates/ingress.yaml

```diff
@@ -1,38 +1,38 @@
 {{- if .Values.ingress.enabled -}}
-{{- $fullName := include "kubeapps.fullname" . -}}
-{{- $ingressPath := .Values.ingress.path -}}
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: {{ $fullName }}
+  name: {{ template "kubeapps.fullname" $ }}
   labels:
-    app: {{ include "kubeapps.name" . }}
-    chart: {{ include "kubeapps.chart" . }}
-    release: {{ .Release.Name }}
-    heritage: {{ .Release.Service }}
-{{- with .Values.ingress.annotations }}
+    app: {{ template "kubeapps.name" $ }}
+    chart: {{ template "kubeapps.chart" $ }}
+    release: {{ $.Release.Name }}
+    heritage: {{ $.Release.Service }}
   annotations:
```

Comment:
note that this key `annotations` in this new case will be always added regardless of we passing `.Values.ingress.annotations` or  `.Values.ingress.certManager ` is that ok?

Reply:
It's fine. The result is: no annotations